bundles/c/c_lexer.moon

```diff
@@ -62,7 +55,6 @@ howl.aux.lpeg_lexer ->
 
   include_stmt = sequence {
     c('preproc', '#include'),
-    ws^1,
```

Comment:
@nilnor It's intentional. It's technically valid and makes reading APL-style C code a little easier. :)